storage/remote/queue_manager.go

```diff
@@ -425,7 +389,6 @@ func (t *QueueManager) updateShardsLoop() {
 }
 
 func (t *QueueManager) calculateDesiredShards() {
-	t.samplesIn.tick()
```

Comment:
Is this the fix for the issue?

Reply:
This is a correctness fix; we shouldn't be ticking this N times, one for each queue.  This isn't the main fix though.